.github/workflows/unittests.yml

```diff
@@ -0,0 +1,28 @@
+name: Operator Unit tests
+
+on:
```

Comment:
Can this just be `on: [push, pull_request]`?  The unit tests are quick, so it should get run on any push or PR.